assets/js/components/notifications/BannerNotifications.test.js

```diff
@@ -30,16 +30,52 @@ import {
 	provideUserInfo,
 	render,
 } from '../../../../tests/js/test-utils';
+import { MODULES_ADSENSE } from '../../modules/adsense/datastore/constants';
 import { CORE_SITE } from '../../googlesitekit/datastore/site/constants';
 import { CORE_USER } from '../../googlesitekit/datastore/user/constants';
+import { CORE_MODULES } from '../../googlesitekit/modules/datastore/constants';
 import { mockLocation } from '../../../../tests/js/mock-browser-utils';
 import BannerNotifications from './BannerNotifications';
+import Header from '../Header';
 
 describe( 'BannerNotifications', () => {
 	mockLocation();
 
 	let registry;
 
+	const testNotification = {
+		id: 'test-notification-id',
+		title: 'Test notification title',
+		content: 'Test notification content',
+		ctaURL: '',
+		ctaLabel: '',
+		ctaTarget: '',
+		dismissible: true,
+		dismissLabel: 'test dismiss site notification',
+		learnMoreURL: '',
+		learnMoreLabel: '',
+	};
+
+	const activateAdsenseModule = () => {
+		muteFetch(
+			new RegExp( '^/google-site-kit/v1/modules/adsense/data/settings' ),
+			[]
+		);
```

Comment:
We don't have to mute the API call since we are mocking the `setSettings` action with the `accountID` below.